sycl/include/sycl/ext/oneapi/matrix/matrix.hpp

```diff
@@ -29,3 +29,6 @@
 #if (SYCL_EXT_ONEAPI_MATRIX_VERSION == 3)
 #include <sycl/ext/oneapi/matrix/matrix-tensorcore.hpp>
 #endif // SYCL_EXT_ONEAPI_MATRIX_VERSION
+#if (SYCL_EXT_ONEAPI_MATRIX_VERSION == 4)
```

Comment:
move unified to use 2.
Remove use implementation .
We need to wait for https://github.com/intel/llvm/pull/7077to get merged first